locales/en/plugin__odf-console.json

```
@@ -47,6 +47,35 @@
   "DRPolicies": "DRPolicies",
   "No disaster recovery policies yet": "No disaster recovery policies yet",
   "Configure recovery to your failover cluster with a disaster recovery policy.<1></1>Click the <3>Create DRPolicy</3> button to get started.": "Configure recovery to your failover cluster with a disaster recovery policy.<1></1>Click the <3>Create DRPolicy</3> button to get started.",
+  "Select": "Select",
+  "Cannot initiate {{actionType}} as DR is not enabled, apply DRPolicy to the application to proceed.": "Cannot initiate {{actionType}} as DR is not enabled, apply DRPolicy to the application to proceed.",
+  "Application name:": "Application name:",
+  "Select policy": "Select policy",
+  "Target cluster": "Target cluster",
+  "Select subscriptions group": "Select subscriptions group",
+  "{{actionType}} initiated": "{{actionType}} initiated",
+  "Initiate": "Initiate",
+  "Intiating": "Intiating",
+  "Failover application": "Failover application",
+  "All system workloads and their available resources will be transferred back to the recovery (failover) site.": "All system workloads and their available resources will be transferred back to the recovery (failover) site.",
```

```suggestion
  "All system workloads and their available resources will be transferred to the recovery (failover) site.": "All system workloads and their available resources will be transferred to the recovery (failover) site.",
```
does transfer "back" to recovery site make sense ?? transfer back to origin sounds good, but is it same for this case as well ??

---

or this origin and recovery sites are relative ?? any one cluster can act as origin and other will be recovery site ??

---

yes, one cluster is the origin and another one is the recovery site. Once the user is done with Failover to the recovery site, they will relocate the workload back to the origin cluster when the origin cluster is back to a healthy state.